packages/stryker-mocha-runner/src/StrykerMochaReporter.ts

```diff
@@ -1,5 +1,5 @@
 import { RunResult, RunStatus, TestStatus } from 'stryker-api/test_runner';
-import * as log4js from 'log4js';
+import * as log4js from 'stryker-api/logging';
```

Comment:
we call it log4js here and import the entire module

Reply:
Good point. Refactored.